tests/e2e/performance/test_fio_benchmark.py

```diff
@@ -308,22 +360,309 @@ def test_fio_workload_simple(self, ripsaw, es, interface, io_pattern):
         except IOError:
             log.info("FIO failed to complete")
 
-        # Clean up fio benchmark
-        log.info("Deleting FIO benchmark")
-        fio_cr_obj.delete()
+        return end_time
 
-        log.debug(f"Full results is : {full_results.results}")
+    def init_full_results(self, full_results):
+        """
+        Initialize the full results object which will send to the ES server
+
+        Args:
+            full_results (obj): an empty FIOResultsAnalyse object
+
+        Returns:
+            FIOResultsAnalyse (obj): the input object fill with data
+
+        """
+        for key in self.environment:
+            full_results.add_key(key, self.environment[key])
+
+        # Setting the global parameters of the test
+        full_results.add_key("dataset", f"{self.total_data_set}GiB")
+        full_results.add_key(
+            "file_size", self.fio_cr["spec"]["workload"]["args"]["filesize"]
+        )
+        full_results.add_key(
+            "servers", self.fio_cr["spec"]["workload"]["args"]["servers"]
+        )
+        full_results.add_key(
+            "samples", self.fio_cr["spec"]["workload"]["args"]["samples"]
+        )
+        full_results.add_key(
+            "operations", self.fio_cr["spec"]["workload"]["args"]["jobs"]
+        )
+        full_results.add_key(
+            "block_sizes", self.fio_cr["spec"]["workload"]["args"]["bs"]
+        )
+        full_results.add_key(
+            "io_depth", self.fio_cr["spec"]["workload"]["args"]["iodepth"]
+        )
+        full_results.add_key("jobs", self.fio_cr["spec"]["workload"]["args"]["numjobs"])
+        full_results.add_key(
+            "runtime",
+            {
+                "read": self.fio_cr["spec"]["workload"]["args"]["read_runtime"],
+                "write": self.fio_cr["spec"]["workload"]["args"]["write_runtime"],
+            },
+        )
+        full_results.add_key(
+            "storageclass", self.fio_cr["spec"]["workload"]["args"]["storageclass"]
+        )
+        full_results.add_key(
+            "vol_size", self.fio_cr["spec"]["workload"]["args"]["storagesize"]
+        )
+        return full_results
+
+    def copy_es_data(self, es, full_results):
+        """
+        Copy data from Internal ES (if exists) to the main ES
+
+        Args:
+            es (obj): elasticsearch object (if exits)
+            full_results (obj): the full results object
 
-        # if Internal ES is exists, Copy all data from the Internal to main ES
+        """
         if es:
             log.info("Copy all data from Internal ES to Main ES")
             es._copy(full_results.es)
         # Adding this sleep between the copy and the analyzing of the results
         # since sometimes the results of the read (just after write) are empty
-        time.sleep(30)
+        time.sleep(10)
+
+    def calculate_compression_ratio(self, pool_name):
```

Comment:
this could be helpful in other places, so consider moving it somewhere under ocs_ci/ lib. 
Also, please try to merge it with ocs_ci.ocs.cluster.validate_compression

Reply:
@ebenahar , regarding moving the function, you are right, i will move it.
regarding merge the two functions, this is a problem, since the calculation is different.

Reply:
Just move this function to cluster.py
@ebenahar , do you think the PR Validation need to run again ? all tests passed before the function moved.